api/src/db.js

```diff
@@ -150,7 +150,18 @@ if (UNIT_TEST_ENV) {
       return [];
     }
 
-    const results = await db.bulkDocs(docs);
+    let results;
+    try {
+      results = await db.bulkDocs(docs);
+    } catch (err) {
+      if (err.status !== 413 || docs.length === 1) {
+        throw err;
+      }
+
+      results = await Promise.all(docs.map(doc => db.put(doc)));
+      results = results.flat();
+    }
```

Comment:
Nit: Let's pull this out as a private function so we can `return results` rather than overwriting, eg:

```
    try {
      return await db.bulkDocs(docs);
    } catch (err) {
      if (err.status !== 413 || docs.length === 1) {
        throw err;
      }
    }
    const results = await Promise.all(docs.map(doc => db.put(doc)));
    return results.flat();
```

Reply:
Nit indulged.